src/TaxonomyAccessUserForm.php

```diff
@@ -31,11 +41,20 @@ public function getFormId() {
     return 'tac_lite_user';
   }
 
-  public function __construct($uid) {
+  /**
+   * @return Connection
+   */
+  public function getDatabase()
+  {
+    return $this->database;
+  }
+
+  public function __construct($uid, $database) {
     $this->account = User::load($uid);
     if (!$this->account) {
       throw new NotFoundHttpException();
     }
+    $this->database = $database;
```

Comment:
I think we don't need the database here right? TermStorage::loadTree does the job.